sunbeam-python/sunbeam/core/deployment.py

```diff
@@ -400,3 +400,7 @@ def get_tfhelper(self, tfplan: str) -> TerraformHelper:
     def get_space(self, network: Networks) -> str:
         """Get space associated to network."""
         return NotImplemented
+
+    def get_spaces(self, networks: list[Networks]) -> set:
```

Comment:
```suggestion
    def get_spaces(self, networks: list[Networks]) -> set[str]:
```

Can we add a bit more typing: set[str]

Reply:
This code chunk is removed in the subsequent push